webui/react/src/themes.ts

```diff
@@ -1,3 +1,6 @@
+import {
+  Determinedjobv1State,
```

Comment:
**question:** not sure how hard it is, but would it be possible to rename these API objects to be something more like `ApiV1JobState`? wondering if it's similar to the work that cut out `determined` prefix from API functions.

Reply:
It'd be nice to do that. I tried to alias these int types.ts but didn't workout for some reason. 
regarding renaming those from the roots It seemed possible but was [hackier than modifying the operation ids](https://github.com/determined-ai/determined/pull/3154#issuecomment-961354983)